src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -421,19 +419,19 @@ public void testFindDataSetsByAuthUserEmail() {
 
         List<Dataset> datasets = datasetDAO.findDatasetsByAuthUserEmail(user.getEmail());
         assertFalse(datasets.isEmpty());
-        List<Integer> datasetIds = datasets.stream().map(Dataset::getDataSetId).collect(Collectors.toList());
+        List<Integer> datasetIds = datasets.stream().map(Dataset::getDataSetId).toList();
         assertTrue(datasetIds.contains(dataset.getDataSetId()));
     }
 
     @Test
     public void testFindNonDACDataSets() {
-        Dataset dataset = createDataset();
+        Dataset dataset = insertDataset();
         Consent consent = createConsent();
```

Comment:
curious: why preference for `insertDataset`? should we start migrating verbage of others, e.g., `createConsent`, into insert as well?